### ISSUE TITLE
Release not being updated

### ISSUE THREAD

**Issue by balloob** [toolmantim/release-drafter#29]
https://github.com/home-assistant/home-assistant-polymer/releases

Our release hasn't been updated in 2 days but PRs have been merged. By any chance any issues in the release drafter logs?

![image](https://user-images.githubusercontent.com/1444314/43248038-e1f69d4e-90b6-11e8-925c-c57855d66b6b.png)

And here are the commits:

![image](https://user-images.githubusercontent.com/1444314/43248055-ec327ff8-90b6-11e8-8bc2-60d432a76c97.png)

**Comment by ludeeus** [comment]
I have the same issue.
It looks like it **only** add it _if_ the branch that are merged are created from the target, after the last PR that got added to the release draft.

Not really good if there are multiple users working on seperate branches.
It might work to rebase before adding a PR, I have not tested that yet.

**Comment by toolmantim** [comment]
Did you manually cut the release @balloob or did it end up working?

@ludeeus I just released a new update which should fix things for you (#30)

**Comment by balloob** [comment]
I manually wrote the release notes 😭

**Comment by toolmantim** [comment]
Boo! That’s no good. I’ll add some logging and let you know what’s up @balloob

**Comment by toolmantim** [comment]
@balloob I've rolled out more logging in #33 so I can search logs for your repo specifically — let me know when you've pushed some changes and I'll see what's up

**Comment by balloob** [comment]
Just merged https://github.com/home-assistant/home-assistant-polymer/pull/1537 and it did not show up in https://github.com/home-assistant/home-assistant-polymer/releases/tag/untagged-b390e522d0986874f2eb

**Comment by toolmantim** [comment]
```
2018-08-02T16:03:31.037Z  INFO probot: home-assistant/home-assistant: Found 12 releases
2018-08-02T16:03:31.057Z  INFO probot: home-assistant/home-assistant: No draft release found
2018-08-02T16:03:31.077Z  INFO probot: home-assistant/home-assistant: Last release: 20180802.0
2018-08-02T16:03:31.097Z  INFO probot: home-assistant/home-assistant: Comparing commits 20180802.0..master
2018-08-02T16:03:32.287Z  INFO probot: home-assistant/home-assistant: Found pull request numbers: 1529
2018-08-02T16:03:32.980Z  INFO probot: home-assistant/home-assistant: Creating new draft release
2018-08-02T19:48:55.203Z  INFO probot: home-assistant/home-assistant: Found 119 releases
2018-08-02T19:48:55.224Z  INFO probot: home-assistant/home-assistant: Draft release: untagged-61b4c47f2d4bed6fca38
2018-08-02T19:48:55.245Z  INFO probot: home-assistant/home-assistant: Last release: 121
2018-08-02T19:48:55.266Z  INFO probot: home-assistant/home-assistant: Comparing commits 121..dev
2018-08-02T19:48:56.752Z  INFO probot: home-assistant/home-assistant: Found pull request numbers: 
2018-08-02T19:48:56.774Z  INFO probot: home-assistant/home-assistant: Updating existing draft release
2018-08-02T19:48:56.716Z  INFO probot: home-assistant/home-assistant: Ignoring push. fix-unicode-error is not one of: dev
2018-08-02T21:18:29.344Z  INFO probot: home-assistant/home-assistant: Ignoring push. uvloop is not one of: dev
2018-08-02T21:18:44.640Z  INFO probot: home-assistant/home-assistant: Found 119 releases
2018-08-02T21:18:44.660Z  INFO probot: home-assistant/home-assistant: Draft release: untagged-61b4c47f2d4bed6fca38
2018-08-02T21:18:44.679Z  INFO probot: home-assistant/home-assistant: Last release: 121
2018-08-02T21:18:44.698Z  INFO probot: home-assistant/home-assistant: Comparing commits 121..dev
2018-08-02T21:18:45.588Z  INFO probot: home-assistant/home-assistant: Found pull request numbers: 614
2018-08-02T21:18:46.579Z  INFO probot: home-assistant/home-assistant: Updating existing draft release
2018-08-02T21:18:53.795Z  INFO probot: home-assistant/home-assistant: Ignoring push. uvloop is not one of: dev
2018-08-02T21:39:45.808Z  INFO probot: home-assistant/home-assistant: Ignoring push. master is not one of: dev
2018-08-02T21:40:57.032Z  INFO probot: home-assistant/home-assistant: Found 119 releases
2018-08-02T21:40:57.053Z  INFO probot: home-assistant/home-assistant: Draft release: untagged-61b4c47f2d4bed6fca38
2018-08-02T21:40:57.074Z  INFO probot: home-assistant/home-assistant: Last release: 121
2018-08-02T21:40:57.095Z  INFO probot: home-assistant/home-assistant: Comparing commits 121..dev
2018-08-02T21:40:58.287Z  INFO probot: home-assistant/home-assistant: Found pull request numbers: 614
2018-08-02T21:40:59.028Z  INFO probot: home-assistant/home-assistant: Updating existing draft release
2018-08-02T21:43:22.175Z  INFO probot: home-assistant/home-assistant: Ignoring push. refs/tags/122 is not one of: dev
2018-08-03T09:23:44.636Z  INFO probot: home-assistant/home-assistant: Found 13 releases
2018-08-03T09:23:44.657Z  INFO probot: home-assistant/home-assistant: Draft release: null
2018-08-03T09:23:44.677Z  INFO probot: home-assistant/home-assistant: Last release: 20180802.0
2018-08-03T09:23:44.701Z  INFO probot: home-assistant/home-assistant: Comparing commits 20180802.0..master
2018-08-03T09:23:52.195Z  INFO probot: home-assistant/home-assistant: Found pull request numbers: 1529
2018-08-03T09:23:57.907Z  INFO probot: home-assistant/home-assistant: Updating existing draft release
2018-08-03T11:03:32.430Z  INFO probot: home-assistant/home-assistant: Found 13 releases
2018-08-03T11:03:32.472Z  INFO probot: home-assistant/home-assistant: Draft release: untagged-b390e522d0986874f2eb
2018-08-03T11:03:32.513Z  INFO probot: home-assistant/home-assistant: Last release: 20180802.0
2018-08-03T11:03:32.552Z  INFO probot: home-assistant/home-assistant: Comparing commits 20180802.0..master
2018-08-03T11:03:33.725Z  INFO probot: home-assistant/home-assistant: Found pull request numbers: 1529
2018-08-03T11:03:34.520Z  INFO probot: home-assistant/home-assistant: Updating existing draft release
2018-08-03T11:31:18.346Z  INFO probot: home-assistant/home-assistant: Found 13 releases
2018-08-03T11:31:18.366Z  INFO probot: home-assistant/home-assistant: Draft release: untagged-b390e522d0986874f2eb
2018-08-03T11:31:18.387Z  INFO probot: home-assistant/home-assistant: Last release: 20180802.0
2018-08-03T11:31:18.407Z  INFO probot: home-assistant/home-assistant: Comparing commits 20180802.0..master
2018-08-03T11:31:19.387Z  INFO probot: home-assistant/home-assistant: Found pull request numbers: 1529
2018-08-03T11:31:20.356Z  INFO probot: home-assistant/home-assistant: Updating existing draft release
2018-08-03T11:37:17.119Z  INFO probot: home-assistant/home-assistant: Found 13 releases
2018-08-03T11:37:17.139Z  INFO probot: home-assistant/home-assistant: Draft release: untagged-b390e522d0986874f2eb
2018-08-03T11:37:17.159Z  INFO probot: home-assistant/home-assistant: Last release: 20180802.0
2018-08-03T11:37:17.180Z  INFO probot: home-assistant/home-assistant: Comparing commits 20180802.0..master
2018-08-03T11:37:18.298Z  INFO probot: home-assistant/home-assistant: Found pull request numbers: 1529
2018-08-03T11:37:19.016Z  INFO probot: home-assistant/home-assistant: Updating existing draft release
2018-08-03T11:37:20.788Z  INFO probot: home-assistant/home-assistant: Ignoring push. refs/tags/20180803.0 is not one of: master
2018-08-03T22:39:51.431Z  INFO probot: home-assistant/home-assistant: Ignoring push. small-image is not one of: dev
2018-08-03T22:41:19.320Z  INFO probot: home-assistant/home-assistant: Found 119 releases
2018-08-03T22:41:19.341Z  INFO probot: home-assistant/home-assistant: No draft release found
2018-08-03T22:41:19.362Z  INFO probot: home-assistant/home-assistant: Last release: 122
2018-08-03T22:41:19.383Z  INFO probot: home-assistant/home-assistant: Comparing commits 122..dev
2018-08-03T22:41:19.974Z  INFO probot: home-assistant/home-assistant: Found pull request numbers: 617
2018-08-03T22:41:20.992Z  INFO probot: home-assistant/home-assistant: Creating new draft release
2018-08-03T22:41:25.807Z  INFO probot: home-assistant/home-assistant: Ignoring push. small-image is not one of: dev
2018-08-03T23:25:57.802Z  INFO probot: home-assistant/home-assistant: Ignoring push. master is not one of: dev
2018-08-03T23:27:05.576Z  INFO probot: home-assistant/home-assistant: Ignoring push. refs/tags/123 is not one of: dev
2018-08-04T13:04:21.979Z  INFO probot: home-assistant/home-assistant: Ignoring push. fix-custom-panel is not one of: master
2018-08-04T13:05:47.400Z  INFO probot: home-assistant/home-assistant: Found 13 releases
2018-08-04T13:05:47.422Z  INFO probot: home-assistant/home-assistant: No draft release found
2018-08-04T13:05:47.445Z  INFO probot: home-assistant/home-assistant: Last release: 20180803.0
2018-08-04T13:05:47.467Z  INFO probot: home-assistant/home-assistant: Comparing commits 20180803.0..master
2018-08-04T13:05:48.953Z  INFO probot: home-assistant/home-assistant: Found pull request numbers: 1538
2018-08-04T13:05:49.716Z  INFO probot: home-assistant/home-assistant: Creating new draft release
2018-08-04T13:15:47.204Z  INFO probot: home-assistant/home-assistant: Found 14 releases
2018-08-04T13:15:47.225Z  INFO probot: home-assistant/home-assistant: Draft release: null
2018-08-04T13:15:47.245Z  INFO probot: home-assistant/home-assistant: Last release: 20180803.0
2018-08-04T13:15:47.266Z  INFO probot: home-assistant/home-assistant: Comparing commits 20180803.0..master
2018-08-04T13:15:48.173Z  INFO probot: home-assistant/home-assistant: Found pull request numbers: 1538, 1539
2018-08-04T13:15:48.194Z  INFO probot: home-assistant/home-assistant: Ignoring push. fix-custom-panel is not one of: master
2018-08-04T13:15:50.252Z  INFO probot: home-assistant/home-assistant: Updating existing draft release
2018-08-04T13:16:50.299Z  INFO probot: home-assistant/home-assistant: Found 14 releases
2018-08-04T13:16:50.320Z  INFO probot: home-assistant/home-assistant: Draft release: untagged-7ba397070ad072986dbf
2018-08-04T13:16:50.342Z  INFO probot: home-assistant/home-assistant: Last release: 20180803.0
2018-08-04T13:16:50.365Z  INFO probot: home-assistant/home-assistant: Comparing commits 20180803.0..master
2018-08-04T13:16:50.996Z  INFO probot: home-assistant/home-assistant: Found pull request numbers: 1538, 1539
2018-08-04T13:16:53.022Z  INFO probot: home-assistant/home-assistant: Updating existing draft release
2018-08-04T13:17:06.030Z  INFO probot: home-assistant/home-assistant: Ignoring push. refs/tags/20180804.0 is not one of: master
2018-08-04T23:19:36.366Z  INFO probot: home-assistant/home-assistant: Ignoring push. voluptuous is not one of: dev
2018-08-06T08:40:35.557Z  INFO probot: home-assistant/home-assistant: Found 14 releases
2018-08-06T08:40:35.580Z  INFO probot: home-assistant/home-assistant: No draft release found
2018-08-06T08:40:35.602Z  INFO probot: home-assistant/home-assistant: Last release: 20180804.0
2018-08-06T08:40:35.624Z  INFO probot: home-assistant/home-assistant: Comparing commits 20180804.0..master
2018-08-06T08:40:36.186Z  INFO probot: home-assistant/home-assistant: Found pull request numbers: 1541
2018-08-06T08:40:37.478Z  INFO probot: home-assistant/home-assistant: Creating new draft release
```

🤔

Can you spot anything weird there @balloob?

**Comment by balloob** [comment]
This is the log from when 1537 got merged:

```
2018-08-03T11:03:32.430Z  INFO probot: home-assistant/home-assistant: Found 13 releases
2018-08-03T11:03:32.472Z  INFO probot: home-assistant/home-assistant: Draft release: untagged-b390e522d0986874f2eb
2018-08-03T11:03:32.513Z  INFO probot: home-assistant/home-assistant: Last release: 20180802.0
2018-08-03T11:03:32.552Z  INFO probot: home-assistant/home-assistant: Comparing commits 20180802.0..master
2018-08-03T11:03:33.725Z  INFO probot: home-assistant/home-assistant: Found pull request numbers: 1529
2018-08-03T11:03:34.520Z  INFO probot: home-assistant/home-assistant: Updating existing draft release
```

Somehow it doesn't find the last pull requests:

![image](https://user-images.githubusercontent.com/1444314/43762242-b219a48e-9a27-11e8-8485-739bc5141707.png)

**Comment by balloob** [comment]
Wrote some tests and looks like pull request extraction always works. So it must be upstream, maybe the commits we pass in?

**Comment by balloob** [comment]
Oh! I was doing a git log to simulate getting the commits and I think I know what it is.

![image](https://user-images.githubusercontent.com/1444314/43763138-42fbd060-9a2a-11e8-981b-05c955c891f3.png)


When we squash and merge, it puts all the original commit messages in the commit message too. It's just that GitHub only shows the first line.

**Comment by balloob** [comment]
Fix in #38.
